compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -620,14 +620,26 @@ void addCPULinalgExtTileAndVectorizePipeline(
     OpPassManager &funcPassManager, TilingConfig &tilingConfig,
     LLVMCPUPipelineOptions &pipelineOpt) {
   addTileAndDistributePasses(funcPassManager);
-  funcPassManager.addPass(
-      createLLVMCPUTilePass(tilingConfig.getVectorCommonParallelLevel()));
-  // TODO: Remove the pass once we have PartialReductionOpInterface implemented
-  // for AttentionOp.
+
+  {
+    LLVMCPUTileRootAndFuseProducerConsumerPassOptions options;
+    options.tilingLevel = tilingConfig.getVectorCommonParallelLevel();
+    options.onlyFuseProducerInputOperands = false;
+    funcPassManager.addPass(
+        createLLVMCPUTileRootAndFuseProducerConsumerPass(options));
+  }
+
   funcPassManager.addPass(
       IREE::LinalgExt::createConvertAttentionToOnlineAttentionPass());
-  funcPassManager.addPass(
-      createLLVMCPUTilePass(tilingConfig.getVectorReductionLevel()));
+
+  {
+    LLVMCPUTileRootAndFuseProducerConsumerPassOptions options;
+    options.tilingLevel = tilingConfig.getVectorReductionLevel();
+    options.onlyFuseProducerInputOperands = true;
+    funcPassManager.addPass(
```

Comment:
`funcPassManager.addPass(createLLVMCPUTileRootAndFuseInputOperands(tilingConfig.getVectorReductionLevel()))`

Reply:
I prefer the options being listed one by one, it's more readable